lms/djangoapps/discussion_api/api.py

```diff
@@ -76,17 +78,35 @@ def get_module_sort_key(module):
     }
 
 
-def _cc_thread_to_api_thread(thread):
+def _cc_thread_to_api_thread(thread, cc_user, staff_user_ids, ta_user_ids, group_ids_to_names):
     """
     Convert a thread data dict from the comment_client format (which is a direct
     representation of the format returned by the comments service) to the format
     used in this API
+
+    Arguments:
+    * thread (comment_client.thread.Thread): The thread to convert
+    * cc_user (comment_client.user.User): The comment_client representation of
+        the requesting user
+    * staff_user_ids (set): The set of user ids for users with the Moderator or
+        Administrator role in the course
+    * ta_user_ids (set): The set of user ids for users with the Community TA
+        role in the course
+    * group_ids_to_names (dict): A mapping of group ids to names
```

Comment:
@gwprice thanks for adding this; in order to be "google style" (per [this](https://github.com/edx/edx-platform/wiki/Python-Guidelines) guideline) i think the indentation needs to be adjusted.